docs/pages/guides/customizing-metro.mdx

```diff
@@ -196,12 +125,15 @@ In the future, this will work universally across platforms with EAS Update hosti
 
 > Available in SDK 49 and higher.
 
-Expo's Metro config has experimental support for the `compilerOptions.paths` and `compilerOptions.baseUrl` fields in the project's `tsconfig.json` (or `jsconfig.json`) file. This enables absolute imports and aliases in the project. Learn more in the [TypeScript guide](/guides/typescript).
+Expo's Metro config has support for the `compilerOptions.paths` and `compilerOptions.baseUrl` fields in the project's `tsconfig.json` (or `jsconfig.json`) file. This enables absolute imports and aliases in the project. Learn more in the [TypeScript guide](/guides/typescript).
```

Comment:
```suggestion
Expo's Metro config supports the `compilerOptions.paths` and `compilerOptions.baseUrl` fields in the project's **tsconfig.json** (or **jsconfig.json**) file. This enables absolute imports and aliases in the project. See [TypeScript](/guides/typescript) guide for more information.
```

- Let's be concise in the first sentence
- Use bold filename pattern as per our writing styleguide
- Improve verbiage around linking to another guide
-